core/templates/app.constants.ts

```diff
@@ -230,5 +230,9 @@ export const AppConstants = {
       DISPLAY_NAME: 'Question Reviews'
     }
   },
-  CONTRIBUTOR_BADGE_INITIAL_LEVELS: [1, 10, 20, 50, 100, 200, 300, 400, 500]
+  CONTRIBUTOR_BADGE_INITIAL_LEVELS: [1, 10, 20, 50, 100, 200, 300, 400, 500],
+
+  INTERACTION_NAME: {
```

Comment:
```suggestion
  INTERACTION_NAMES: {
```

Reply:
another object exists with "INTERACTION_NAMES" so I gues this name would be fine?

Reply:
I'm concerned about this constant in general. Why do we have it here and why do we need it now when we didn't before?

A generic constant like "INTERACTION_NAME(S)" should be including names for all interactions. But you really shouldn't need this because the interaction specs already contain a human-readable description of the interaction. Having this constant is just duplicating information which is not a great practice from a design perspective.